### ISSUE TITLE
Aurora drop in calendar wrong img format 

### ISSUE THREAD

**Issue by exezbcz** [kodadot/nft-gallery#10080]
![image](https://github.com/kodadot/nft-gallery/assets/90852205/29527bbd-5a68-4790-bb43-2efa91f01d99)

it looks like the image is not a square. 

![image](https://github.com/kodadot/nft-gallery/assets/90852205/0b9f3f3b-cd37-4107-8dac-1639cd2d1069)

cc @JustLuuuu

**Comment by JustLuuuu** [comment]
connected with artist

**Comment by ThomasssArt** [comment]
Are you guys 100% sure that you use the right file? Can you open the sketch.js and check line 24 if its exactly the same than here?!
![image](https://github.com/kodadot/nft-gallery/assets/60002234/d44469b8-c6e8-4a62-9418-abb5f2e3d626)

**Comment by JustLuuuu** [comment]
yeah Thomas is right, in the corrected file this shouldn't be an issue.

**Comment by ThomasssArt** [comment]
Here again the project. I just renamed the zip file. Can you try this?

[Aurora_final.zip](https://github.com/kodadot/nft-gallery/files/14994378/Aurora_final.zip)
